src/Dapper.Logging/Hooks/ISqlHooks.cs

```diff
@@ -8,5 +8,6 @@ public interface ISqlHooks<in T>
         void ConnectionOpened(DbConnection connection, T context, long elapsedMs);
         void ConnectionClosed(DbConnection connection, T context, long elapsedMs);
         void CommandExecuted(DbCommand command, T context, long elapsedMs);
+        void CommandFailed(DbCommand command, T context, object elapsedMs, Exception exception);
```

Comment:
Adding a method to an interface is a breaking change.
Should we have an empty body here using the _default interface members_ language feature? Also, using an `object` for elapsedMs seems like an error.

Reply:
I've completely forgotten that the default interface members require `net standard 2.1` and increasing it is also a breaking change.